### ISSUE TITLE
Icon: some icons are very wide

### ISSUE THREAD

**Issue by elabowski** [brewkit/components#78]
Looking through the storybook icon list, a few icons are more than 20px wide, with hundreds of px of whitespace. The signal_wifi icons the ones I want to use. See attached image.
![Screen Shot 2020-02-28 at 1 39 26 PM](https://user-images.githubusercontent.com/52831050/75577554-37d72c80-5a30-11ea-8d62-b3502a7891d0.png)

**Comment by lawren** [comment]
Looks like this is an issue with the actual google webfont maintained separately. We may need to consider adopting Material UI's icon format using SVGs...

**Comment by lawren** [comment]
Resolved as of version 2.2.0.

Please refer to the [Brewkit docs](https://brewkit.github.io/components/?path=/docs/stable-display-icons-general--page) for usage info on the new Icon format.

Essentially, you'll want to instead lean on the icons from `@material-ui/icons` which is a dependency of Brewkit and use those instead.

```tsx
import { SignalWifi0BarOutlinedIcon } from '@material-ui/icons';

(): React.ReactElement => (
    <div>
        <SignalWifi0BarOutlinedIcon />
    </div>
)
```
